Silicon/CommonSocPkg/Library/HeciLib/HeciCore.c

```diff
@@ -988,3 +989,38 @@ HeciServiceResetInterface (
 {
   return HeciResetInterface((HECI_DEVICE)HeciDev);
 }
+
+/**
+  Check for Manufacturing Mode
+
+  @param[out]  MeManuMode         Manufacturing mode/production mode.
+
+  @retval EFI_UNSUPPORTED         Current CSME mode doesn't support this function
+  @retval EFI_SUCCESS             Command succeeded
+
+**/
+EFI_STATUS
+EFIAPI
+HeciGetManufactureMode (

```

Comment:
This API should be moved to HeciLib.c instead of HeciCore.c

Reply:
Done